### ISSUE TITLE
Cant't interact with linux while this is running.

### ISSUE THREAD

**Issue by sand7000** [belvederef/visual-snow-relief-overlay#17]
Running up to date archlinux with fluxbox.

The overlay intercepts all mouse clicks and keystrokes making the system unusable.

Also I had to install xcompmgr to get this to have transparency, otherwise I couldn't see anything except snow while it was running. Might want to add that to the linux instructions.

**Comment by peterrus** [comment]
@sand7000 did you minimize the application? That should let you interact with your system again.

**Comment by belvederef** [comment]
> Running up to date archlinux with fluxbox.
> 
> The overlay intercepts all mouse clicks and keystrokes making the system unusable.
> 
> Also I had to install xcompmgr to get this to have transparency, otherwise I couldn't see anything except snow while it was running. Might want to add that to the linux instructions.

@sand7000 It does not apply to all linux systems I believe. I tested on Ubuntu and Manjaro it just works out of the box. If this gets brought up again we can add it to the docs, yours could have been an isolated case.

**Comment by sergioramalli** [comment]
I am currently using Ubuntu 22.04.2 LTS and i am facing a few issues. The overlay doesn't allow me to interact with Terminal or my code editor but it will work for google chrome. 

I had an older version of Ubuntu, (18 I believe) and it was working fine. After the upgrade I starting have these issues.

**Comment by ravanamerongen** [comment]
I am having the same issues on Ubuntu 220.04.2 LTS

**Comment by LostNulide** [comment]
I don’t think this application will work on newer distributions because Wayland currently prevents applications from allowing input passthrough, at least to my knowledge. I attempted to develop a solution myself but found that it requires closer integration with the compositor. As a result, I created an extension specifically for GNOME(as this is the de i am using), which leverages some additional features.

You can check it out at https://gitlab.com/Nulide/gsnow.

I hope it will be available soon on the [GNOME Extensions](https://extensions.gnome.org/) site. :)
